src/PostTypes/Course.php

```diff
@@ -58,7 +58,7 @@ class Course
             ),
             'admwpp_tms_legacy_id' => array(
                 'type' => 'text',
-                'label' => 'Legacy ID',
+                'label' => 'LegacyID',
```

Comment:
Can you please rebase on lates trunk this change is a roll back of when we merged yesterday.